op-proposer/proposer/driver.go

```diff
@@ -266,7 +329,6 @@ func (l *L2OutputSubmitter) waitForL1Head(ctx context.Context, blockNum uint64)
 			if err != nil {
 				return err
 			}
-			break
```

Comment:
Is the changed behavior by not breaking here any more covered by a test?

Reply:
Shouldn't be! This is a redundant break on the switch case, caught by the go linter.